tests/cmake/emscripten_version/CMakeLists.txt

```diff
@@ -1,16 +1,22 @@
-# EMSCRIPTEN_VERSION CMake variable is introduced in Emscripten version 1.38.6, so test that the value is at least that new.
+project(emscripten_version)
```

Comment:
what does this do?

Reply:
It just suppresses the warning that cmake prints if its missing:

```
CMake Warning (dev) in CMakeLists.txt:
  No project() command is present.  The top-level CMakeLists.txt file must
  contain a literal, direct call to the project() command.  Add a line of
  code such as

    project(ProjectName)

  near the top of the file, but after cmake_minimum_required().

  CMake is pretending there is a "project(Project)" command on the first
```